internal/host/statemachine.go

```diff
@@ -403,9 +403,9 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler) sta
 	sm.AddTransition(stateswitch.TransitionRule{
 		TransitionType: TransitionTypeRefresh,
 		SourceStates: []stateswitch.State{
-			stateswitch.State(models.HostStatusPreparingSuccessful),
+			stateswitch.State(models.HostStatusPreparingFailed),
```

Comment:
We still need the transition from HostStatusPreparingSuccessful.

Reply:
there is  one for it 
`	sm.AddTransition(stateswitch.TransitionRule{
		TransitionType: TransitionTypeResetHost,
		SourceStates: []stateswitch.State{
			stateswitch.State(models.HostStatusPreparingForInstallation),
			stateswitch.State(models.HostStatusPreparingSuccessful),
		},
		DestinationState: stateswitch.State(models.HostStatusKnown),
		PostTransition:   th.PostResetHost,
	})`

Reply:
This is reset, not refresh

Reply:
got it , fixing